components/common/ListTable/ListTable.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { composeStories } from '@storybook/react'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import * as stories from './ListTable.stories'
+
+const { Common } = composeStories(stories)
+
+function setup() {
+  const user = userEvent.setup()
+  render(<Common {...Common.args} />)
+  return { user }
+}
+
+describe('[component] - ListTable', () => {
+  it('should render the list table', () => {
+    setup()
+    const table = screen.getByRole('table')
+    expect(table).toBeVisible()
+  })
+  it('should check for number of rows in list table', () => {
+    setup()
+    const tableRows = screen.getAllByRole('row')
+    if (Common.args) {
+      // adding one to length because using TableRow in the TableHead
+      expect(tableRows.length).toEqual(Common.args.rows.length + 1)
+    }
+  })
+})
```

Comment:
Instead of checking number of rows, we should check whether all the rows are rendered or not along with 'Initiate Quote', 'Add To Cart' link and edit, duplicate and delete icons.

Missing Testcases
1. should redirect user to Quote page when user clicks on 'Initiate Quote' link
2. should add list in Cart when user clicks on 'Add To Cart' link
3. should redirect user to Edit Quote page when user clicks on 'Edit' icon
4. should redirect user to Duplicate Quote page when user clicks on 'Duplicate' icon
5. should show Confirmation dialog when user clicks on 'Delete' icon